index.html

```diff
@@ -36,5 +36,135 @@ <h3>LET'S CONNECT</h3>
                 <a href="#"><li><img src="./Vector (2).png" alt="victory"></li></a>
             </ul>
     </section>
+<div class="grid-container1">
+       <div class="grid-item1"><img src="Snapshoot Portfolio.png" alt="Snapshoot portfolio"></div>
+       <div class="grid-item2">
+        <h2>Tonic</h2>
+        <ul>
+            <li style="list-style: none;" id="canopy">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p>A daily selection of privately<br>
+             personalized reads; no accounts or<br>
+              sign-ups required.</p>
+              <ul style="list-style: none;" class="language-button">
+                <button type="submit"><li>html</li></button> 
+              <button type="submit"><li>css</li></button>
+                <button type="submit"><li>javascript</li></button>
+             </ul>
```

Comment:
- Good job using a `ul` tag, but I'd suggest you wrap the `buttons` in a `li` tag instead of the other way round. Remembering this will help improve your site's accessibility and HTML structure 👍 👍. Please fix this for all your project cards
- Also, your design doesn't exactly match the design given in the Figma template. Remember it is important to replicate the Figma template as closely as possible to give recruiters confidence in your ability to replicate exact business requirements 👍 . Please fix this for all your project cards

Your design(pay attention to the background color, border, border-radius and spacing between the individual buttons):
![project tech student](https://user-images.githubusercontent.com/73607512/193030549-5251e9c8-215c-4e6a-8329-e25dcf4dad38.PNG)

Figma template:
![project tech](https://user-images.githubusercontent.com/73607512/193030633-8e7e7cb6-e72f-4cf1-8aa5-44c5113b8cf5.PNG)